src/styles/login.css

```diff
@@ -0,0 +1,296 @@
+}
+.landing>.login-wrapper>form>.login-form-header>h1{
+    font-size:6rem;
+    color:#0080ff;
+  
+}
+.landing>.login-wrapper>form>.login-form-header>h2{
+    font-size: 2.5rem;
+    font-weight: 500;
+    padding:1rem;
+    margin: 3rem 2rem;
+}
+.form-group{
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    margin: 1rem auto;
+    width: 90%;
+}
+.form-group> label{
+    font-size:1.6rem;
+    font-weight: 500;
+    padding-bottom: 1rem;
+    padding-top: 2.5rem;
+}
+.form-group .form-control{
+    font-size: 1.6rem;
+    padding:1.5rem 1rem;
+    border-radius: 0.5rem;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+    border:none;
+}
+.btn-primary{
+    color: white;
+    font-size: 2rem;
+    padding: 1rem;
+    cursor: pointer;
+    background: #0080ff;
+    border:none;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+    border-radius: 0.5rem;
+    font-weight: 600;
+    margin-top: 3rem;
+}
+.btn-primary:hover{
+    background: white;
+    color: #0080ff;
+    transition: 0.5s;
+}
+.login-wrapper>.footer h4{
+    font-size: 1.6rem;
+    font-weight: 200;
+    padding-top: 1rem;
+    margin-bottom: 3rem;
+}
+.blue-span{
+    color:#0080ff;
+    font-weight: 500;
+}
+.login-sidbar>.sidebar-heading {
+    margin: 0 4rem;
+    width: 100%;
+}
+.login-sidbar>.sidebar-heading >h1{
+    font-family: 'Gothic A1', sans-serif;
+    font-weight: 600;
+    font-size: 6rem;
+    line-height: 8rem;
+    padding-right: 3rem;
+    padding-left: 3rem;
+    letter-spacing: 0.3rem;
+    
+}
+.login-sidbar>.sidebar-heading >h2{
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: normal;
+    font-size: 4rem;
+    line-height: 4.7rem;
+    margin: 2rem auto;
+}
+.sidebar-step-items{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width:90%;
+    margin: 0 auto;
+}
+.step-number{
+    font-size: 12rem;
+    color: rgba( 0, 0, 0, 0.1);
+    margin:1rem;
+  
+}
+.step-text{
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    justify-content:center;
+   font-size: 1.4rem;
+   font-family: "roboto";
+   flex: 1;
+   
+}
+.step-text p{
+    color: gray;
+}
+.step-text>h3{
+    font-size: 2rem;
+    padding-bottom: 1rem;
+    font-weight: 600;
+    letter-spacing: 0.1rem;
+    color:black;
+}
+.step-svg{
+    display: flex;
+    flex: 1;
+    height:17rem;
+    width: 30rem; 
+
+}
+.step-svg img{
+   width: 100%;
+}
+.green-span::before{
+    /* color: #57B894; */
+    color: #0080ff;
+    content:'';
+    animation: animate infinite 10s;
+}
+
+
+@keyframes animate {
+    0%{
+        content: 'Connect';
+        
+    }
+    20%{
+        content: 'Survive';
+    }
+    40%{
+        content: 'Grow';
+    }
+    60%{
+        content: 'Succeed';
+    }
+    80%{
+        content: 'Mature';
+    }
+    100%{
+        content: 'Connect';
+    }
+}
+.signup-sub{
+    padding:0 !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)